bahmni-docker/README.md

```diff
@@ -161,6 +161,16 @@ The below steps needs to performed only once after OpenMRS application is loaded
 3. In that page click on `Rebuild Search Index`
 4. This rebuilds concept index of OpenMRS application.
 
+# One-time Setup for Crater:
+If you don't wish to do let crater do automatic installation, you can set the `CRATER_AUTO_INSTALL` to `"false"` an follow the below steps:
+1. Open your web browser and go to your given domain (http://crater.test) and follow the installation wizard.
```

Comment:
The URL would be `http://localhost:81`

Reply:
The URLs are updated in all the instances.